EIPS/eip-698.md

```diff
@@ -0,0 +1,98 @@
+## Preamble
+    
+	EIP: 698
+ 
+	Title: OPCODE 0x46 BLOCKREWARD
+    
+	Author: Cody Burns <dontPanic@codywburns.com>
+    
+	Type: Standard Track
+    
+	Category: Core
+    
+	Status: Draft
+    
+	Created: 2017-28-08
+
+
+
+## Simple Summary
+
+This EIP adds an additional opcode to the EVM which will return a finalized blocks reward value. 
+
+
+
+## Abstract
+
+In the EVM, the 0x40 opcodes are reserved for `Block Information`. Currently reserved opcodes are:
+* `0X40 BLOCKHASH`
+* `0X41 COINBASE`
+* `0X42 TIMESTAMP`
+* `0X43 NUMBER`
+* `0X44 DIFFICULTY`
+* `0X45 GASLIMIT`
+
+This EIP would add an additional opcode, `0x46 BLOCKREWARD`, which would return the block reward for any finalized block. The finalized block reward would include the base reward, uncle payments, and gas.   
+
+## Motivation
+
+
+Per EIP-649 ( #669 ) periodic block reward reductions/variance are now planned in the roadmap, however, this EIP is consensus system agnostic and is most useful in decentralized pool operations and for any contract that benefits from knowing a block reward payout(i.e. Merge mined tokens) 
+
+
+## Specification
+
+
+After block `n` all clients should process opcode `0x46` as follows:  
+
+* Value: `0x46`
+* Mnemonic: `BLOCKREWARD`
+* δ:` 0` nothing removed from stack
+* α:`1` block reward added to stack
```

Comment:
Since the total reward cannot be known for the current block, it may be more useful to:
- have an input parameter (block number)
- return 4 outputs: base reward, uncle reward, variable (gas) reward (as proposed here https://github.com/ethereum/EIPs/issues/698#issuecomment-325156613) - maybe even outputting total gas consumed

It may be possible to implement this change similar to how #210 is done, that would make it clear that all this detail can only be retrieved for past blocks.